pkg/processor/build/builder_test.go

```diff
@@ -581,6 +668,86 @@ func (suite *testSuite) mergeDirectivesAndVerify(first map[string][]functionconf
 
 }
 
+func (suite *testSuite) testResolveFunctionPathRemoteCodeFile(fileExtension string) {
+
+	// mock http response
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+
+	codeFileContent:= "some code..."
+	responder := func(req *http.Request) (*http.Response, error) {
+		responder := httpmock.NewStringResponder(200, codeFileContent)
+		response, err := responder(req)
+		response.ContentLength = int64(len(codeFileContent))
+		return response, err
+	}
+	pythonFileURL := "http://some-address.com/my-func."+fileExtension
+	httpmock.RegisterResponder("GET", pythonFileURL, responder)
+
+	// the python file will be downloaded here
```

Comment:
downloaded? what do you mean?

Reply:
`resolveFunctionPath`  downloads the code file it gets as parameter when it's a URL, so this downloaded file will be saved inside the temp dir we create there

Reply:
updated comment to: 
`// the archive will be "downloaded" to this directory`